src/app/pages/home/home.jsx

```diff
@@ -0,0 +1,123 @@
+
+/**
+ * Dependencies.
+ */
+
+var React = require('react');
+var StoreMixin = require('fluxible').StoreMixin;
+var GoogleMapsStore = require('../../maps/google-maps-store');
+var NavLink = require('../../nav/nav-link');
+var loadGoogleMaps = require('../../maps/load-google-maps');
+
+/**
+ * Home page.
+ */
+
+module.exports = React.createClass({
+  mixins: [StoreMixin],
+  statics: {
+    storeListeners: [GoogleMapsStore]
+  },
+  getInitialState: function () {
+    var venues = this.props.context.getStore('venue').getVenues();
```

Comment:
I read about that antipattern already. It seems like it doesn't apply very much to this case. If I was defining state based on data passed in through props, it would make sense. The rationale is against having multiple sources of truth. However, that's not the case here. Props is used to pass down the context which is then used as the single source of truth. The context isn't really data, it's access. So I don't think that argument applies to this situation.